File: Iniciante/1008 - Salário/bee1008.c
// Beecrowd 1008 - Salário
#include <stdio.h>

int main() {
    int n,h;
    double sal,sal_liq;

    scanf("%d", &n);
    scanf("%d", &h);
    scanf("%lf", &sal);


    sal_liq = sal*h;

    printf("NUMBER = %d\n", n);
    printf("SALARY = U$ %.2lf\n", sal_liq);
    return 0;
}

File: Iniciante/1008 - Salário/bee1008.java
// Beecrowd 1008 - Salário
import java.util.Scanner;

public class bee1008 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int n = scan.nextInt();
        int Hrs = scan.nextInt();
        double Sal = scan.nextDouble();
        double total = Hrs*Sal;
        System.out.println("NUMBER = " + n);
        System.out.printf("SALARY = U$ %.2f%n", total);

    }
}
